nightshift/marc/marc_parser.py

```diff
@@ -8,19 +8,49 @@
 evideo)
 """
 from io import BytesIO
+import logging
 import pickle
 from typing import Any, BinaryIO, Iterator, Optional, Union
 
 from bookops_marc import SierraBibReader, Bib
+from pymarc import parse_xml_to_array, Record
```

Comment:
`Record` object is what `worldcat_response_to_pymarc()` returns. The import is there because of typing:
```python
def worldcat_response_to_pymarc(response: bytes) -> Record:
    """
    Converts MetadataApi responses into `pymarc.Record` objects.

    Args:
        response:                           MARC XML in binary format

    Returns:
        `pymarc.Record` instance
```